src/core/auth.ts

```diff
@@ -84,9 +85,12 @@ export interface AuthSDK {
    * Authenticates user with email and password.
    *
    * @param opts - Object with user's email and password.
+   * @param includeDetails - Whether to fetch user details. Default is true.
```

Comment:
includeDetails description should not be described as separate param, instead as param in opts param